applications/dashboard/models/class.usermodel.php

```diff
@@ -4065,6 +4064,59 @@ public function saveAuthentication($data) {
      * @throws Exception
      */
     public function setCalculatedFields(&$user) {
+        if (is_object($user)) {
+            $this->setCalculatedFieldsObject($user);
+        } else {
```

Comment:
I'd have done a `return` at the end of the `if` to avoid putting the entire method in an `else` block. Would have made it cleaner when the time comes to extract that deprecated method.